src/components/Auth/Signup.js

```diff
@@ -35,6 +38,7 @@ const Signup = () => {
         });
         localStorage.setItem("token", response.data.encodedToken);
         localStorage.setItem("user", JSON.stringify(response.data));
+
         navigateHome("/");
```

Comment:
Check here also for response.status, if it is fine then only send your dispatch otherwise don't.